railties/lib/rails/configuration.rb

```diff
@@ -131,6 +131,15 @@ def after_generate(&block)
         @after_generate_callbacks << block
       end
 
+      def apply_autocorrect_after_generate!
```

Comment:
I'd go with `apply_rubocop_autocorrect_after_generate!`. Make it clear that it's rubocop specific.

Reply:
Hmm, don't actually love having this inside this class. This is a very generic class. We need like a helper level place to put it.

Reply:
Actually it's fine. This is specific to generators.

Reply:
This is a development note. Initially, considering future expansions like in https://github.com/rubygems/rubygems/pull/4411, I thought it would be better to have an abstract naming like the `--linter` option in Bundler. However, if this feature is limiting it to rubocop-rails-omakase, so `apply_rubocop_autocorrect_after_generate` seems to have been a polished clear name. Thank you for updating it!